www/src/components/shell/onboarding/checklist/Footer.tsx

```diff
@@ -3,17 +3,16 @@ import { Button, Flex } from 'honorable'
 import { useContext } from 'react'
 
 import { OnboardingChecklistContext } from '../../../../contexts/OnboardingChecklistContext'
-
 import { UPDATE_USER } from '../../../users/queries'
-import { ONBOARDING_CHECKLIST_STATE } from '../../constants'
+import { updateUserFragment } from '../../../../utils/graphql'
+import { RootMutationType, RootMutationTypeUpdateUserArgs } from '../../../../generated/graphql'
```

Comment:
We should check if we can setup tslint in a way that this would be using non-relative path. If it's more that one level it's getting hard to read.

Reply:
It would be best to add aliases for top directories in tscompiler and vite.